web_src/js/index.js

```diff
@@ -1259,7 +1259,13 @@ function initPullRequestReview() {
       td.find("input[name='side']").val(side === 'left' ? 'previous' : 'proposed');
       td.find("input[name='path']").val(path);
     }
-    commentCloud.find('textarea').focus();
+    const $textarea = commentCloud.find('textarea');
+    attachTribute($textarea.get(), {mentions: true, emoji: true});
+
+    // Give new write/preview data-tab name to distinguish from others
+    const $simplemde = setCommentSimpleMDE($textarea);
+    $textarea.focus();
```

Comment:
What's this focus for?

Reply:
it's copying the same code as the ` $('.edit-content').on('click', async function (event) {...}`

Reply:
Seems like a noop to me unless the textarea focus triggers something, but whatever.

Reply:
I agree it's likely a no-op - but's it's present elsewhere and so I think it's prudent to put it there in case it's there for a reason. I don't want to introduce a new heisenbug.